server/middleware/tokenVerification.js

```diff
@@ -18,7 +19,7 @@ const isLoggedin = (req, res, next) => {
   }
 
   try {
-    const decodedToken = jwt.verify(token, keys);
+    const decodedToken = jwt.verify(token, process.env.tokenKeys);
```

Comment:
It's best practice to name your variables in the dotenv file in a capital case format and use an underscore to separate words in this instance TOKEN_KEY.

Reply:
Done